src/main/java/seedu/realodex/model/util/SampleDataUtil.java

```diff
@@ -29,47 +30,53 @@ public static Person[] getSamplePersons() {
                        new Address("Blk 30 Geylang Street 29, #06-40"),
                        new Family("20"),
                        getTagSet("Buyer"),
-                       new Remark("Has 3 cats.")),
+                       new Remark("Has 3 cats."),
+                       new Birthday("")),
             new Person(new Name("Bernice Yu"),
                        new Phone("99272758"),
                        new Income("1000"),
                        new Email("berniceyu" + "@example.com"),
                        new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"),
                        new Family("20"),
                        getTagSet("buyer", "seller"),
-                       new Remark("Eats alot")),
+                       new Remark("Eats alot"),
+                       new Birthday("01-Apr-2021")),
```

Comment:
Thanks for adding 2 people with the same birthday, it will help in testing the sort function when sorting by birthdays in future